### ISSUE TITLE
Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/accessibility/AccessibilityNodeInfo$ExtraRenderingInfo;

### ISSUE THREAD

**Issue by topcoco** [mozilla/rhino#1829]
![Image](https://github.com/user-attachments/assets/e942288a-c8a1-43c9-9078-6fc35f66feec)

**Comment by topcoco** [comment]
﻿
My platform is Android. When I call a function from JavaScript, it reports an error, but when I directly call it in Java, there is no problem. Both Rhino versions 1.7.15 and 1.7.16 encounter the same error, which is caused by:
﻿
Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/accessibility/AccessibilityNodeInfo$ExtraRenderingInfo;
Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.accessibility.AccessibilityNodeInfo$ExtraRenderingInfo"
This error indicates that the ExtraRenderingInfo inner class of AccessibilityNodeInfo in the Android framework is not found when the JavaScript code attempts to invoke the related functionality.
